pkg/registry/pod/etcd/etcd_test.go

```diff
@@ -99,6 +99,11 @@ func TestCreate(t *testing.T) {
 	test := resttest.New(t, storage, fakeEtcdClient.SetError)
 	pod := validNewPod()
 	pod.ObjectMeta = api.ObjectMeta{}
+	invalidPod := validNewPod()
+	invalidPod.Namespace = test.TestNamespace()
+	invalidPod.Labels = map[string]string{
+		"invalid/label/to/cause/validation/failure": "bar",
+	}
 	test.TestCreate(
 		// valid
 		pod,
```

Comment:
Can't comment on lines you didn't change... Can you also add a comment why the pod below is invalid. My guess is that it's because it has empty container list, but that should be obvious.